docs-beta/src/components/Illustration.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export const Illustration = (props) => {
+  const imageStyle = `mx-auto ${
+    props.isWave ? "w-full h-full max-h-40" : "w-3/4 h-3/4 md:w-full md:h-full"
```

Comment:
Tailwind is still not generating css for the max-h-40. :( I say we leave it in, and track as a bug for future.